app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -242,11 +219,14 @@ class SearchActivity : AppCompatActivity(), ClickListenerForRecyclerView {
     }
 
     private fun hideHistoryLayout() {
-        historyLayout.visibility = View.GONE
+        binding.historyLayout.visibility = View.GONE
     }
 
     override fun onClick(track: Track) {
         searchHistory.addTrack(track)
+        val displayIntent = Intent(this, PlayerActivity()::class.java)
```

Comment:
```suggestion
        val displayIntent = Intent(this, PlayerActivity::class.java)
```
Целый объект создавать тут не нужно